docs/podman-generate-systemd.1.md

```diff
@@ -27,35 +30,74 @@ Set the systemd restart policy.  The restart-policy must be one of: "no", "on-su
+
 [Unit]
 Description=c21da63c4783be2ac2cd3487ef8d2ec15ee2a28f63dd8f145e3b05607f31cffc Podman Container
+
 [Service]
-Restart=on-failure
+Restart=always
 ExecStart=/usr/bin/podman start c21da63c4783be2ac2cd3487ef8d2ec15ee2a28f63dd8f145e3b05607f31cffc
-ExecStop=/usr/bin/podman stop -t 10 c21da63c4783be2ac2cd3487ef8d2ec15ee2a28f63dd8f145e3b05607f31cffc
+ExecStop=/usr/bin/podman stop -t 1 c21da63c4783be2ac2cd3487ef8d2ec15ee2a28f63dd8f145e3b05607f31cffc
 KillMode=none
 Type=forking
 PIDFile=/var/run/containers/storage/overlay-containers/c21da63c4783be2ac2cd3487ef8d2ec15ee2a28f63dd8f145e3b05607f31cffc/userdata/conmon.pid
+
 [Install]
 WantedBy=multi-user.target
 ```
 
-Create a systemd unit file for a container running nginx with an *always* restart policy and 1-second timeout.
+Create systemd unit files for a pod with a simple alpine container.
 ```
-$ sudo podman generate systemd --restart-policy=always -t 1 nginx
+$ podman pod create --name systemd-pod
+$ podman create --pod systemd-pod alpine top
+$ podman generate systemd --name systemd-pod
+/home/user/pod-systemd-pod.service
+/home/user/container-admiring_visvesvaraya.service
+```
+
+Create systemd unit files on stdout for a pod with a simple alpine container on stdout.
+```
+$ podman pod create --name systemd-pod
+$ podman create --pod systemd-pod alpine top
+$ podman generate systemd --name systemd-pod
+# pod-systemd-pod.service
 [Unit]
-Description=c21da63c4783be2ac2cd3487ef8d2ec15ee2a28f63dd8f145e3b05607f31cffc Podman Container
+Description=systemd-pod Podman Pod
+Documentation=man:podman-generate-systemd(1)
+Requires=container-admiring_visvesvaraya.service
+Before=container-admiring_visvesvaraya.service
+
 [Service]
-Restart=always
-ExecStart=/usr/bin/podman start c21da63c4783be2ac2cd3487ef8d2ec15ee2a28f63dd8f145e3b05607f31cffc
-ExecStop=/usr/bin/podman stop -t 1 c21da63c4783be2ac2cd3487ef8d2ec15ee2a28f63dd8f145e3b05607f31cffc
+Restart=on-failure
+ExecStart=/home/valentin/go/src/github.com/containers/libpod/bin/podman pod start systemd-pod
+ExecStop=/home/valentin/go/src/github.com/containers/libpod/bin/podman pod stop  systemd-pod
 KillMode=none
 Type=forking
-PIDFile=/var/run/containers/storage/overlay-containers/c21da63c4783be2ac2cd3487ef8d2ec15ee2a28f63dd8f145e3b05607f31cffc/userdata/conmon.pid
+# PID of the infra container's conmon
+PIDFile=/tmp/1000/overlay-containers/fc26bf24717105a3e7d77ad3c926f71562a2a60e28cde5bc325dd11de75bc663/userdata/conmon.pid
+
+[Install]
+WantedBy=multi-user.target
+
+# container-admiring_visvesvaraya.service
+[Unit]
+Description=admiring_visvesvaraya Podman Container
+Documentation=man:podman-generate-systemd(1)
+PartOf=pod-systemd-pod.service
+
+[Service]
+Restart=on-failure
+ExecStart=/home/valentin/go/src/github.com/containers/libpod/bin/podman start admiring_visvesvaraya
+ExecStop=/home/valentin/go/src/github.com/containers/libpod/bin/podman stop -t 10 admiring_visvesvaraya
```

Comment:
Should we be adding a timeout here? We're overriding whatever the user-specified timeout during creation was.

Reply:
This behaviour pre-dates this PR. We're only overriding if the user sets the timeout explicitly via `generate systemd --timeout`.